Source/MLX/Documentation.docc/Articles/converting-python.md

```diff
@@ -26,6 +26,8 @@ Python code | Swift Code
 `array[..., None]` | `array.expandedDimensions(axis: -1)`
 `array[:, -1, :]` | `array[-1, axis: 1]`
 `array[..., ::2]` | `array[stride: 2, axis: -1]`
+`array[::-1]` | `array[stride: -1, axis: 0]` -- reverse first dimension of array
+`array[..., ::-1]` | `array[stride: -1, axis: -1]` or `array[stride: -1]` -- reverse last dimension of array
```

Comment:
These are the patterns we see

<img width="1026" alt="image" src="https://github.com/ml-explore/mlx-swift/assets/46639364/bb12afb3-7193-46c5-ae81-654672d0d77d">